README.md

```diff
@@ -175,6 +175,17 @@ REM stop the service
 sc.exe stop DeadlineWorker
 ```
 
+### Running Outside of an Operating System Service
+
+The worker agent can also be started outside of a service context if required. Run `deadline-worker-agent --help` to see a list of supported command line arguments.
```

Comment:
Can we add a warning indicate that using the OS service is the preferred approach for production use. The benefits of the OS service are:

1.  it restarts the worker agent process if it crashes
2. handles starting the agent when the host machine boots
3. handles attempts to gracefully shutting down the agent when the host machine is shutting down

Reply:
Added a warning with this information above